src/pages/index.js

```diff
@@ -146,22 +168,34 @@ export function updateProfile(profileObject) {
       name: newName,
       occupation: newOccupation,
     })
+    .then(() => {
+      userInfo.setUserInfo(newName, newOccupation);
+      editProfileFormValidator.disableSubmitButton();
+      editProfileModal.close();
+    })
     .finally(() => {
       profileSubmitButton.textContent = "Save";
+    })
+    .catch((error) => {
+      console.error("Oops, something happened", error);
     });
-  userInfo.setUserInfo(newName, newOccupation);
-  editProfileFormValidator.disableSubmitButton();
-  editProfileModal.close();
 }
 
 export function updateProfilePicture(profileObject) {
   const newPhotoLink = profileObject["input-url"];
   console.log(profileObject);
   changeAvatarSubmitButton.textContent = "Saving...";
-  api.updateAvatar(newPhotoLink).then((response) => {
-    userInfo.setUserAvatar(response.avatar);
-    changeAvatarSubmitButton.textContent = "Save";
-  });
+  api
+    .updateAvatar(newPhotoLink)
+    .then((response) => {
+      userInfo.setUserAvatar(response.avatar);
+    })
+    .finally(() => {
+      changeAvatarSubmitButton.textContent = "Save";
+    })
+    .catch((error) => {
+      console.error("Oops, something happened", error);
+    });
   updateAvatarFormValidator.disableSubmitButton();
```

Comment:
`.disableSubmitButton` is called when you open the modal. So, you don't need it here